build.gradle

```diff
@@ -70,6 +70,7 @@ subprojects {
 
             tasks.withType(ScalaCompile) {
                 scalaCompileOptions.additionalParameters = gradle.scala.compileFlags
+                scalaCompileOptions.forkOptions.jvmArgs = ["-Xss2m"]
```

Comment:
Is this required to update the pure config?

Reply:
It used to for 0.12 yes. Compilation would yield StackOverflowErrors otherwise. I can try to back it out again though to see if it now works without that.

Reply:
`StackOverflowError`s in https://travis-ci.org/apache/openwhisk/jobs/618509410